workflow/tests/test_signals.py

```diff
@@ -102,6 +103,21 @@ def test_demo_workflowteam_assignment_not_reassigned_on_update(
         self.assertEqual(num_results, 1)
 
 
+@tag('pkg')
+class CreateDefaultProgramTest(TestCase):
```

Comment:
Create a new test case to make sure that you're not creating programs when you update the org.

Reply:
Stupid me, true. I just fixed it